litebird_sim/bandpasses.py

```diff
@@ -0,0 +1,275 @@
+from dataclasses import dataclass
+
+import numpy as np
+from scipy import signal
+
+from typing import Union
+from uuid import UUID
+
+from .imo import Imo
+
+
+@dataclass
+class BandPassInfo(object):
+
+    """A class wrapping the basic information about a detector bandpass.
+    This  class   encodes the basic properties of a frequency band.
+    It can be initialized in three ways:
+    - Through the default constructor, will assume top-hat bandpasses
+    provided the band centroid and the width are issued.
+    - Through the class method :meth:`.from_imo`, which reads the
+      detector bandpass from the LiteBIRD Instrument Model (see
+      the :class:`.Imo` class).
+
+    Args:
+        - nu_center (float):  center frequency in GHz
```

Comment:
We use the convention that measure units are appended to the arguments/variable members rather than just specified in the code. In this way, the following code is clear even without resorting to the documentation:

```python
band = BandPassInfo(nu_center_ghz=30.0)  # Instead of nu_center=30.0
```

Reply:
ok i agree!